assets/scripts/components/codetabs.js

```diff
@@ -223,7 +310,24 @@ const initCodeTabs = () => {
         })
     }
 
+    /**
+     * If Cdocs is running on this page,
+     * tell it to refresh the tabs when content changes
+     */
+    if (window.clientFiltersManager) {
+        // Update the tabs after the page is initially rendered
+        // and before it is revealed
```

Comment:
```suggestion
        // and revealed
```
Or just delete the line entirely